docs/feature_custom_modified_values.md

```diff
@@ -0,0 +1,444 @@
+# CMV (Custom Modified Values)
```

Comment:
You need to add a link to `_summary.md` and `_sidebar.md` for this page to show up in docs correctly.